Oracle/README.md

```diff
@@ -171,29 +203,49 @@ These files will contain the definitions of the objects specified by the file na
 
 ### Insufficient JVM memory
 
-If you encounter "insufficient memory" errors while executing the script, you can uncomment the following line in the `create_ddls.sh` or `create_ddls_plus.sh` file to allocate 4 GB of RAM: `export JAVA_TOOL_OPTIONS=-Xmx4G`. You can modify this parameter by changing the number on the line to allocate a different amount of memory.
+1. If you encounter "insufficient memory" errors while executing the script, you can uncomment the following line in the `create_ddls.sh` or `create_ddls_plus.sh` file to allocate 4 GB of RAM: `export JAVA_TOOL_OPTIONS=-Xmx4G`. You can modify this parameter by changing the number on the line to allocate a different amount of memory.
+
+2. If you encounter a memory heap error while working with the script, it may be helpful to adjust the values of the LONGCHUNKSIZE variable. This variable determines the size of memory chunks allocated for handling long data in SQL statements. By modifying the LONGCHUNKSIZE value, you can potentially address the memory issue.
+If you are using SQLcl, you can make the necessary changes in the `create_ddls.sql` file. Locate the LONGCHUNKSIZE variable declaration within the file and adjust its value according to your needs. Save the file and rerun the script again.
+If you are using SQLPlus, navigate to the `create_ddls_plus.sql` file. In this file, find the declaration of the LONGCHUNKSIZE variable and modify it as required. Save the file and execute it again.
+By experimenting with different LONGCHUNKSIZE values, you can optimize memory allocation and potentially overcome memory heap errors. It's recommended to test and adjust the value carefully, considering the specific requirements and limitations of your environment.
+
+### Unnecessary blank tab spaces or missing code
+
+If you encounter an issue with unnecessary blank tab spaces or truncated code, you may need to increase the size of the LONGCHUNKSIZE variable found in the create_ddls.sql or create_ddls_plus.sql file, depending on the technology you are using.
+
+Excessive blank tab spaces or truncated code can cause syntax errors or unexpected behavior in your scripts. Adjusting the LONGCHUNKSIZE variable allows for larger memory allocations when handling long data in SQL statements.
+
+To address this issue, locate the LONGCHUNKSIZE variable declaration in the respective file (`create_ddls.sql` for SQLcl or `create_ddls_plus.sql` for SQL\*Plus). Modify the value of LONGCHUNKSIZE to a larger value that suits your requirements. Save the file and rerun your SQL utility.
+
+By increasing the LONGCHUNKSIZE, you provide more memory for handling long data, which can help resolve problems related to unnecessary tab spaces or truncated code. Remember to consider the specific needs and limitations of your environment when adjusting this variable.

```

Comment:
I am just a bit curious about what it means "your specific needs", how can the user know what a correct value for this variable should be after getting these unnecessary blank spaces? What would happen if the user increases the variable to a gigantic value? Would that solve the issue all the time or that would introduce another problem?

Reply:
Maybe we could guide a bit the user to a possible correct value or at least explain what happens if the value is too large

Reply:
there isn't a specify value or usual value. All depends of your PC settings, like ram, PC processor, internet speed ... right now It sets now  with a "Big number" because the default is 80. so, there isn't a recommended value or common value, depends of your computer.

Reply:
I added a little tip to increase and test the value.